website/versioned_docs/version-v1.x/cli.md

```diff
@@ -29,14 +29,14 @@ pnpm run hardhat
+| `create`             | Deploy the contracts                                                     | `-i, --initialVoiceCredits <initialVoiceCredits>`: The initial voice credits <br/> `-p, --initialVoiceCreditsProxyAddress <initialVoiceCreditsProxyAddress>`: The initial voice credits proxy contract address <br/> `-g, --signupGatekeeperAddress <signupGatekeeperAddress>`: The signup gatekeeper contract address <br/> `-ph3, --poseidonT3Address <poseidonT3Address>`: The PoseidonT3 contract address <br/> `-ph4, --poseidonT4Address <poseidonT4Address>`: The PoseidonT4 contract address <br/> `-ph5, --poseidonT5Address <poseidonT5Address>`: The PoseidonT5 contract address <br/> `-ph6, --poseidonT6Address <poseidonT6Address>`: The PoseidonT6 contract address <br/> `-q, --quiet`: Whether to print values to the console <br/> `-s, --stateTreeDepth <stateTreeDepth>`: The state tree depth <br/> `-uq, --use-quadratic-voting"`: Whether to use quadratic voting                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
```

Comment:
I found there is an option called `-r,  --rpc-provider <provider>, the rpc provider URL`  in the cli/index.ts file